### PR TITLE
fix(editor): disable auto save

### PR DESCRIPTION
**Pull request by danilowoz**
This disables the auto save, in order to fix bugs around it, as there are a lot of users making complaints that their changes are suddenly gone.

### CODE REVIEW

**Review by CompuIves — CHANGES_REQUESTED** [review]
Looks very good! Remember to update the `vscode` cache links if you change vscode, then it's good to go!

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5982/commits/2bb09dcb788dce34cf955759d9dd5f10f6671a23 is at https://pr5982.build.csb.dev/s/new.

File: packages/app/src/app/overmind/effects/vscode/constants.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const EXTENSIONS_LOCATION = process.env.VSCODE
   ? '/vscode/extensions-bundle'
-  : '/public/vscode-extensions/v16';
+  : '/public/vscode-extensions/v17';
```
Comment (CompuIves) [inline packages/app/src/app/overmind/effects/vscode/constants.ts:3]: This change is not needed 😄

**Review by CompuIves — APPROVED** [review]
One thing, and then ready to go!

File: packages/app/src/app/overmind/effects/vscode/constants.ts (hunk 1 above)
Comment (CompuIves) [reply packages/app/src/app/overmind/effects/vscode/constants.ts:3]: Because we haven't changed the extensions 😄

**Pull request merged**